.travis.yml

```diff
@@ -1,8 +1,9 @@
 os:
+  - linux
 # TODO: Enable this when we can get the tests to work
 #  - windows
-  - linux
-  - osx
+# Disable while trying helm test on travis.
+#  - osx
```

Comment:
Why the helm test doesn't work on osx?

Reply:
Would have to download mac helm for OSX just to run the test :(

I think i'm going to exclude this test from travis, but keep it in the local pre-commit.

Reply:
That sounds good to me

Reply:
done